src/decider/agent/DeciderAgent.php

```diff
@@ -407,4 +435,33 @@ protected function areNomosMonkNinkaAgreed($licenseMatches)
     }
     return true;
   }
+
+  /**
+   * @brief Check if findings by all agents are same or not
+   * @param LicenseMatch[][] $licenseMatches
+   * @return boolean True if they match, false otherwise
+   */
+  protected function areNomosOrMonkAndOJOAgreed($licenseMatches)
+  {
+    $scanners = array('ojo','monk','nomos');
+    $currentFinding = array();
+    if (array_key_exists('ojo', $licenseMatches) === FALSE) {
+      return false;
+    }
+    foreach ($scanners as $scanner) {
+      if (array_key_exists($scanner, $licenseMatches) === FALSE) {
+        continue;
+      }
+      foreach ($licenseMatches[$scanner] as $licenseMatch) {
+        $licId = $licenseMatch->getLicenseId();
+        if (empty($currentFinding)) {
+          $currentFinding[] = $licId;
+        }
+        if (in_array($licId, $currentFinding) === FALSE) {
+          return false;
+        }
+      }
+    }
+    return true;
+  }
```

Comment:
I think this function can be written in a cleaner way (with two new functions `scannerHasNoMatch` and `getLicenseIdsOfMatchesForScanner`):
```suggestion
  protected function areNomosOrMonkAndOJOAgreed($licenseMatches)
￼  {
    if (scannerHasNoMatch('ojo', $licenseMatches)) {
      return false;
    }
    $findingsByOjo = getLicenseIdsOfMatchesForScanner('ojo', $licenseMatches);
    
    $otherScanners = array('monk','nomos');
    foreach ($otherScanners as $scanner) {
      if (scannerHasNoMatch($scanner, $licenseMatches)) {
        continue;
      }
      $findingsByOtherScanner = getLicenseIdsOfMatchesForScanner($scanner, $licenseMatches);

      // TODO: here needs to be a logic which compares `$findingsByOjo` to `$findingsByOtherScanner`
    }
    return true;
  }
```